packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1164,6 +1166,25 @@ class RenderBackdropFilter extends RenderProxyBox {
     markNeedsPaint();
   }
 
+  /// The blend mode to use to apply the filtered background content onto the background
```

Comment:
looks like this is the same text as above. Use a macro?

Reply:
Done